spec/AutomatedLinkAcceptance+services.yaml

```diff
@@ -1,87 +1,99 @@
-          uuid: xx-1-0-0-op-s-bm-003
+          uuid: ALA-1-0-0-op-s-bm-003
         - service-name: /v1/end-subscription
-          uuid: xx-1-0-0-op-s-bm-004
+          uuid: ALA-1-0-0-op-s-bm-004
         - service-name: /v1/inquire-oam-request-approvals
-          uuid: xx-1-0-0-op-s-bm-005
+          uuid: ALA-1-0-0-op-s-bm-005
         - service-name: /v1/update-client
-          uuid: xx-1-0-0-op-s-bm-007
+          uuid: ALA-1-0-0-op-s-bm-007
         - service-name: /v1/list-ltps-and-fcs
-          uuid: xx-1-0-0-op-s-bm-008
+          uuid: ALA-1-0-0-op-s-bm-008
         - service-name: /v1/redirect-topology-change-information
-          uuid: xx-1-0-0-op-s-bm-009
+          uuid: ALA-1-0-0-op-s-bm-009
         - service-name: /v1/update-operation-key
-          uuid: xx-1-0-0-op-s-bm-010
+          uuid: ALA-1-0-0-op-s-bm-010
         - service-name: /v1/update-operation-client
-          uuid: xx-1-0-0-op-s-bm-011
+          uuid: ALA-1-0-0-op-s-bm-011
       individual:
         - service-name: /v1/bequeath-your-data-and-die
-          uuid: xx-1-0-0-op-s-im-000
+          uuid: ALA-1-0-0-op-s-im-000
     service:
       basic:
+        - service-name: /v1/start-application-in-generic-representation
+          uuid: ALA-1-0-0-op-s-bs-000
         - service-name: /v1/inform-about-application
-          uuid: xx-1-0-0-op-s-bs-001
+          uuid: ALA-1-0-0-op-s-bs-001
         - service-name: /v1/inform-about-application-in-generic-representation
-          uuid: xx-1-0-0-op-s-bs-002
+          uuid: ALA-1-0-0-op-s-bs-002
         - service-name: /v1/inform-about-release-history
-          uuid: xx-1-0-0-op-s-bs-003
+          uuid: ALA-1-0-0-op-s-bs-003
         - service-name: /v1/inform-about-release-history-in-generic-representation
-          uuid: xx-1-0-0-op-s-bs-004
+          uuid: ALA-1-0-0-op-s-bs-004
+        - service-name: /docs
+          uuid: ALA-1-0-0-op-s-bs-005
       individual:
-        - service-name: /v1/start-application-in-generic-representation
-          uuid: xx-1-0-0-op-s-is-000
-        ## TODO: Add Services specific to this application
+        - service-name: /v1/provide-acceptance-data
+          uuid: ALA-1-0-0-op-s-is-000
   http-server:
-    own-application-name: ## TODO: Add OwnApplicationName
-    own-release-number: ## TODO: Add OwnReleaseNumber (e.g. 1.0.0)
-    uuid: xx-1-0-0-http-s-bm-000
-  tpc-server:
-    own-ip-address: ## TODO: Add OwnIpAddress
-    own-tcp-port: ## TODO: Add OwnTcpPort
-    uuid: xx-1-0-0-tcp-s-bm-000
-
+    own-application-name: AutomatedLinkAcceptance
+    own-release-number: 1.0.0
+    uuid: ALA-1-0-0-http-s-000
+  tcp-server:
+    - description: Without TLS layer
+      local-protocol: http
+      own-ip-address: 1.1.4.9
+      own-tcp-port: 4009
+      uuid: ALA-1-0-0-tcp-s-000
+    - description: With TLS layer
+      local-protocol: https
+      own-ip-address: 1.1.4.9
+      own-tcp-port: 4009
+      uuid: ALA-1-0-0-tcp-s-001
 
 
 clients:
 
+
```

Comment:
no ElasticSearch http client found here. In case link acceptance data should be stored inside a database it would be needed, if not than it is correct, that it is missing.
(I assume it is not needed, but maybe it would be good to add a comment here in that case, so that it's clear that it indeed is not needed, rather than just forgotten to add. :) )

From documentation:
```ElasticSearchClients
In case the application is expected to create a lot of internal data, it is better to write it into a database instead of a datafile.
An ElasticSearchClient shall be used for documenting the REST connection to the database.
The latest template of the ServiceList is containing such an ElasticSearchClient.
It just has to be completed by a [Fake Index Alias from the List](https://github.com/openBackhaul/ApplicationPattern/blob/develop/doc/TestingApplications/Infrastructure/SdnLaboratory/FakeAddresses/IndexAliases.md). Potentially, the PlatformOwner has to be asked to add another Fake Index Alias to the list.
If connecting a database would not be required, the ElasticSearchClient has to be deleted from the ServiceList.```

Reply:
It was explicitly specified that no ElasticSearch http client is needed